app/styles/colors-fonts.css

```diff
@@ -0,0 +1,116 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;0,600;1,400;1,500;1,600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Maven+Pro:wght@500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,400;1,500&display=swap');
+
+:root {
+  --AppsilonBlue: #0099f9;
+  --AppsilonNavi: #15354a;
+  --AppsilonIce: #a6d5e0;
+  --AppsilonMint: #1affd5;
+  --AppsilonYellow: #fcd73d;
+  --AppsilonRose: #ff364a;
+  --AppsilonPurple: #770f7c;
+  --AppsilonGray: #6e757b;
+  --AppsilonLightGray: #8a8e91;
+  --AppsilonAsh: #b3b8ba;
+  --AppsilonSnow: #fff;
+  --AppsilonBlue-weak-1: #4cb8fb;
+  --AppsilonBlue-weak-2: #80ccfc;
+  --AppsilonBlue-weak-3: #ccebfe;
+  --AppsilonBlue-strong-1: #0085e1;
+  --AppsilonBlue-strong-2: #0071cd;
+  --AppsilonBlue-strong-3: #005db9;
+  --AppsilonNavi-weak-1: #29495e;
+  --AppsilonNavi-weak-2: #3d5d72;
+  --AppsilonNavi-weak-3: #517186;
+  --AppsilonNavi-strong-1: #0b2b40;
+  --AppsilonNavi-strong-2: #012136;
+  --AppsilonNavi-strong-3: #00172c;
+  --AppsilonIce-weak-1: #b4dce4;
+  --AppsilonIce-weak-2: #c3e3ea;
+  --AppsilonIce-weak-3: #e1f1f4;
+  --AppsilonIce-strong-1: #92c1cc;
```

Comment:
The `colors-fonts.css` seems like overkill as we're not using most of it.
Can we cut down most of it and move it to some scss file instead of CSS?

Reply:
Removed in f56f1cd